ember-cli-build.js

```diff
@@ -85,9 +85,10 @@ module.exports = function(_options) {
   /*
    * ES6 Build
    */
-  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', '..', 'lib');
+  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', '..', 'src');
   // TODO: WAT, why does { } change the output so much....
-  var HTMLTokenizer = find(tokenizerPath, { });
+  var tokenizerTree = find(tokenizerPath, { }), { output: 'tree' });
```

Comment:
I don't think `stew.find` accepts `output` as an option (perhaps this is left over from a prior `stew.log`)?